pkg/scm/scm.go

```diff
@@ -23,8 +23,11 @@ func DownloaderForSource(s string, forceCopy bool) (build.Downloader, string, er
 		return &empty.Noop{}, s, nil
 	}
 
-	details, mods := git.ParseFile(s)
+	details, mods, err := git.ParseFile(s)
 	glog.V(4).Infof("return from ParseFile file exists %v proto specified %v use copy %v", details.FileExists, details.ProtoSpecified, details.UseCopy)
+	if err != nil && !forceCopy {
```

Comment:
this should be checking the type of error that came back.  the only error you can ignore here is an "EmptyGitDir" error, right?